lib/utils.js

```diff
@@ -11,8 +11,9 @@ import {
   trim,
   forEach,
   merge,
-  extend
-} from 'lodash'
+  extend,
+  forOwn
+} = require('lodash')
```

Comment:
sorry to change it back to `require` somehow if i use `import` the unit test will just fail...